src/app/public/modules/autonumeric/autonumeric-config.service.ts

```diff
@@ -0,0 +1,68 @@
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+import { map } from 'rxjs/operators';
+import { Options as AutonumericOptions } from 'autonumeric';
+import { SkyCurrencyFormat, SkyI18nSkyCurrencyFormatService } from './MOCK-currency-format.service';
+
+type IsoCodeAndLocale = { isoCode: string; locale: string };
+
+/**
+ * Service to map a locale + iso code to an Autonumeric Config
+ */
+@Injectable({
+  providedIn: 'root'
+})
+export class SkyAutonumericConfigService {
+  constructor(private currencyFormatService: SkyI18nSkyCurrencyFormatService) { }
+
+  /**
+   * Creates an Autonumeric Config from a Locale and Currency Code.
+   * http://autonumeric.org/guide
+   * @param params optional params object
+   * @param params.isoCode the ISO 4217 Code. Defaults to `USD`.
+   * @param params.locale the Locale. Defaults to `SkyAppLocaleProvider.getLocaleInfo`.
+   */
+  public getAutonumericConfig(params?: Partial<IsoCodeAndLocale>): AutonumericOptions {
```

Comment:
SkyAppLocaleProvider returns an observable (bc users can override it) so I think providing a non-observable-locking call could be valuable.

Defintiely open to opionions though.